as-pect.config.js

```diff
@@ -3,6 +3,9 @@ module.exports = {
 	 * A set of globs passed to the glob package that qualify typescript files for testing.
 	 */
 	include: ['src/as/**/*.spec.ts'],
+	// include: ['src/as/**/Mesh.spec.ts'],
+	// include: ['src/as/**/Geometry.spec.ts'],
+	// include: ['src/as/**/BufferAttribute.spec.ts'],
```

Comment:
I use these commented lines to test a particular file without running all tests. I didn't mean to commit these comments. I'll clean up before the PR is ready.

Reply:
- [ ] cleanup comments